docs/reuse/links.txt

```diff
@@ -16,7 +16,7 @@
 .. _Multipass: https://multipass.run/docs
 .. _`Open Container Initiative`: https://opencontainers.org/
 .. _Rockcraft: https://documentation.ubuntu.com/rockcraft/
-.. _Snap: https://snapcraft.io/docs/installing-snapd
+.. _snapd installed: https://snapcraft.io/docs/installing-snapd
```

Comment:
Can we call this just `_snapd` and do an override where we need the text "snapd installed"?